internal/cli/cmd/dac/build/build.go

```diff
@@ -98,10 +100,13 @@ func (o *option) Execute() error {
 			return nil
 		}
 
-		err = o.processFile(path)
-		if err != nil {
-			// Tell the user about the error but don't stop the processing
-			fmt.Printf("error processing file %s: %v\n", path, err)
+		extension := filepath.Ext(path)
+		if extension == goExtension || extension == cueExtension {
```

Comment:
Why are you checking the extension at this level in the case of directory processing, but not in the case of a single file just above? Given the logic of processFile() I guess you can just remove this check

Reply:
Early check for reducing error spam for unrelated files, example: README, git file, ...

Could be removed but it could produce more noise in log

Reply:
perhaps we can ignore this check and just call directly `processFile` and rely on the function to ensure to execute only file that are known

Reply:
Ok, fair enough